force-app/main/default/classes/IBMWatsonClient.cls

```diff
@@ -35,7 +35,7 @@ public class IBMWatsonClient {
       	String form64 = ((IBMWatsonMultipartBody)request.getBody()).form64();
       	httpRequest.setBodyAsBlob(EncodingUtil.base64Decode(form64));
         httpRequest.setHeader('Content-Length', String.valueof(form64.length()));
-      } else if (request.getBody().contentType == IBMWatsonMediaType.parse('application/json')){
+      } else if (request.getBody().contentType.toString().contains(IBMWatsonHttpMediaType.APPLICATION_JSON)) {
```

Comment:
What is the need for this change?

Reply:
we have `text/plain` as a possible content type for input on the ToneAnalyzer (rather than JSON) see https://watson-api-explorer.mybluemix.net/apis/tone-analyzer-v3#!/tone/tone